nntoolkit/create.py

```diff
@@ -48,18 +48,26 @@ def main(nn_type, architecture, model_file):
 
     filenames = ["model.yml"]  # "input_semantics.csv", "output_semantics.csv"
 
-    if nn_type == 'mlp':
+    layers_binary = []
+    # TODO: the activation function could be here!
+    neurons = list(map(int, architecture.split(':')))
+
+    neurons_a, neurons_b=zip(neurons, neurons[1:])[0]
+ 
+    fan_in=neurons[0]
+    fan_out= len(neurons)-2  
+    model={}
+
+    if nn_type[0] == 'mlp':
```

Comment:
well i printed nn_type=='mlp' and it was false then I printed nn_type, it was a list, so I just did it :P